app/backend/app.py

```diff
@@ -339,6 +360,97 @@ async def speech():
         return jsonify({"error": str(e)}), 500
 
 
+if PYRIT_COMPATIBLE:
+
+    @bp.route("/evaluate", methods=["POST"])
+    @authenticated
+    async def evaluate(auth_claims: dict[str, Any]):
+        request_form = await request.form
+        request_files = await request.files
+        if "input_data" not in request_files:
+            return jsonify({"message": "No input_data part in the request", "status": "failed"}), 400
+        if "config" not in request_form:
+            return jsonify({"message": "No application setting config part in the request", "status": "failed"}), 400
+
+        input_data_file = request_files.get("input_data")
+        num_questions = int(request_form.get("num_questions", 0))
+        config = json.loads(request_form.get("config", "{}"))
+
+        if input_data_file is not None:
+            input_data = [json.loads(line) for line in input_data_file.readlines()]
+        save_jsonl(input_data, Path("./evaluation/input/input_temp.jsonl"))
+        config["testdata_path"] = "input/input_temp.jsonl"
+        config["results_dir"] = "results"
+
+        temp_config_path = Path("config_temp.json")
+
+        save_config(config, "evaluation" / temp_config_path)
```

Comment:
Looks like you would be overwriting the same config file if more than one requests come in. On top of that, it doesn't look like you ever delete it.

Reply:
It looks like the temp file will be loaded instantly after been written so I guess it doesn't really affect the running result in normal situations (these lines look a bit dumb though). They should be some sort of legacy code, I'll handle it.